### PR TITLE
[docs-update][xs]: updated verifying section with example commands - …

### PR DESCRIPTION
**Pull request by Mikanebu**
* rewrote "Verifying" section in the http://datahub.io/docs/getting-started/installing-data so it suggests to do info, get, cat commands with some datahub dataset

### CODE REVIEW

File: docs/getting-started/installing-data.md [hunk 1]
```diff
@@ -38,4 +38,22 @@ This should output a version number, for example on my machine it shows:
 ```
 0.4.1
 ```
+\- To get all the available commands:
+```
+data help
+```
+
+\- Get info on data (file or dataset) at path or url
```
Comment (anuveyatsu) [inline docs/getting-started/installing-data.md]: I think this sentence is incorrect..

Comment (Mikanebu) [reply docs/getting-started/installing-data.md]: @anuveyatsu then we have to fix main help message, it was taken from there

Comment (anuveyatsu) [reply docs/getting-started/installing-data.md]: @Mikanebu sorry, just re-read it and it seems OK. Please, just remove all unnecessary examples and leave only info command.

File: docs/getting-started/installing-data.md [hunk 2]
```diff
@@ -39,3 +39,9 @@ This should output a version number, for example on my machine it shows:
 0.4.1
 ```
 
+\- Get info on data (file or dataset) at path or url
+```bash
+data info http://datahub.io/core/finance-vix
+```
+
+
```
Comment (anuveyatsu) [inline docs/getting-started/installing-data.md]: Can you have a description of the output here? Maybe you can have output in the console. See this: http://datahub.io/docs/getting-started/pushing-data#push-the-data

**Comment by rufuspollock** [comment]
@anuveyatsu @Mikanebu  based on the feedback we had from Brook i would have put this in the getting started section. The install guide should be very simple and checking version is enough to check things are ok.

The info command was a way for people to get a feel for CLI so should be in getting started (and that was what was said on call).

In addition the screenshot that has been added looks really weird -- it is way too big i think. (Who is checking this when live?)

**Pull request merged**